packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -314,13 +288,13 @@ class NewDisk extends Component {
     const diskLimit = this._getResourceSetDiskLimit()
     const resourceSet = this._getResolvedResourceSet()
 
-    const SelectSr_ =
+    const selectSr_ =
```

Comment:
Why?

Reply:
`Imported JSX component SelectSr_ must be in PascalCase.`
These changes are not related to the PR, I'll leave it as before.